src/actions/organization.tsx

```diff
@@ -67,10 +68,10 @@ const createOrganization = ( name: string ) => (dispatch: Dispatch<ICreatedOrgan
 	const organizationPayload = { name }
 	return api.POST('/organizations', organizationPayload)
 		.then((organizationResponse: Response) => organizationResponse.json())
```

Comment:
I'm a bit suprised we're calling `json()` in the actions. Is it not done in the API helper?